cmd/eskip/media.go

```diff
@@ -93,25 +103,38 @@ func validateSelectWrite(cmd command, media []*medium) (input, output *medium, e
 		return nil, nil, missingInput
 	}
 
-	if out == nil {
-		var err error
-		out, err = processEtcdArgs(defaultEtcdUrls, defaultEtcdPrefix)
-		if err != nil {
-			return nil, nil, err
+	return in, out, nil
+}
+
+func validateSelectDelete(media []*medium) (in, out *medium, err error) {
+	if len(media) == 0 {
+		return nil, nil, nil
+	}
+
+	if len(media) > 2 {
+		return nil, nil, tooManyInputs
+	}
+
+	for _, m := range media {
+
+		if m.typ == etcd || m.typ == innkeeper {
+			out = m
+		} else {
+			in = m
 		}
 	}
 
+	if in == nil {
+		return nil, nil, missingInput
+	} else if in.typ != inlineIds {
```

Comment:
that's correct... I got that condition wrong... I will write a test to cover for this and then fix it :)